packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -259,6 +259,11 @@ export const Combobox = <T extends string | number>({
       ? 'search'
       : 'angle-down';
 
+  if (items.length > 100) {
+    const msg = `There are ${items.length} loaded into the component. This exceeds the recommended amount.`;
```

Comment:
```suggestion
    const msg = `[Combobox] Items exceeds the recommended amount ${RECOMMENDED_ITEMS_AMOUNT}.`;
```
We don't want Faro to report separate error messages for all different values. I'm not sure if it manages to pick up the pattern.